source/localizable/tutorial/routes-and-templates.md

```diff
@@ -171,9 +171,34 @@ Now, we'll add a link to our contact page so we can navigate from back and forth
 </div>
 ```
 
+## A Rentals Route
+We want our application to show a list of rentals that users can browse.
+To make this happen we'll add a third route and call it `rentals`.
+
+```shell
+ember g route rentals
+```
+
+Let's update the newly generated `rentals.hbs` with some basic markup to seed our rentals list page.
+We'll come back to this page later to add in the actual rental properties.
+
+```app/templates/rentals.hbs
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Welcome!</h2>
+  <p>We hope you find exactly what you're looking for in a place to stay.</p>
+  {{#link-to 'about' class="button"}}
+    About Us
+  {{/link-to}}
+</div>
+```
+
 ## An Index Route
 
 With our two static pages in place, we are ready to add our home page which welcomes users to the site.
+At this point our main page in our application is our rentals page, which we've already created a route for. 
```

Comment:
Nitpick: ending the sentence with a preposition. `for which we've already created a route.`